fhir/com.b2international.snowowl.fhir.core/src/com/b2international/snowowl/fhir/core/provider/FhirCodeSystemApiProvider.java

```diff
@@ -88,25 +88,25 @@ public FhirCodeSystemApiProvider(IEventBus bus, List<ExtendedLocale> locales) {
 	}
 
 	@Override
-	public boolean isSupported(LogicalId logicalId) {
+	public boolean isSupported(CodeSystemURI codeSystemId) {
 		
 		Optional<String> supportedPath = getSupportedURIs().stream()
 			.map(u -> u.substring(u.lastIndexOf("/") + 1))
-			.filter(p -> p.equals(logicalId.getBranchPath()))
+			.filter(p -> p.equals(codeSystemId.getPath()))
```

Comment:
`CodeSystemURI.getPath()` contains only the version part, which I think won't actually produce any matches here.

Reply:
This actually works for our internal FHIR code systems.